test/simple/test-http-many-keep-alive-connections.js

```diff
@@ -5,7 +5,7 @@ var common = require('../common');
 var assert = require('assert');
 var http = require('http');
 
-var expected = 10000;
+var expected = (process.platform === 'ios') ? 500 : 10000;
```

Comment:
This should be fixed for Android as well, 10000 connections don't make sense on a mobile device regardless of the OS.